swap_meet/clothing.py

```diff
@@ -0,0 +1,11 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "Clothing", condition = 0):
+            self.category = category 
+            self.condition = condition 
```

Comment:
Consider using super and not allowing the user to pass in a category for an instance of `Clothing` since a `Clothing` object should always have the category `"clothing"`
```suggestion
    def __init__(self, condition = 0):
            super().__init__(category='clothing', condition=condition)
```